src/plot.js

```diff
@@ -90,11 +94,15 @@ export function plot(options = {}) {
   }
 
   // Wrap the plot in a figure with a caption, if desired.
-  if (caption == null) return svg;
-  const figure = document.createElement("figure");
-  figure.appendChild(svg);
-  const figcaption = figure.appendChild(document.createElement("figcaption"));
-  figcaption.appendChild(caption instanceof Node ? caption : document.createTextNode(caption));
+  let figure = svg;
+  if (caption != null) {
+    figure = document.createElement("figure");
+    figure.appendChild(svg);
+    const figcaption = figure.appendChild(document.createElement("figcaption"));
+    figcaption.appendChild(caption instanceof Node ? caption : document.createTextNode(caption));
+  }
+
+  figure.scales = exposeScales(scaleDescriptors);
```

Comment:
```suggestion
  figure.scale = exposeScales(scaleDescriptors);
```

Since this returns a single scale now.